code/modules/mob/living/sneeze.dm

```diff
@@ -0,0 +1,49 @@
+/// Launch a sneeze that can infect with a disease
+/mob/living/proc/infectious_sneeze(datum/disease/disease, force, range = 4, charge_time = 0.5 SECONDS, obj/projectile/sneezoid = /obj/projectile/sneeze)
+	sneeze(range, charge_time, sneezoid, on_sneeze_hit_callback = CALLBACK(src, PROC_REF(try_sneeze_infect), disease, force))
+
+/// Try and infect following a sneeze hit. force to always infect
+/mob/living/proc/try_sneeze_infect(datum/disease/disease, force, mob/living/target)
+	target.AirborneContractDisease(disease, force)
+
+/// Inhale and start the sneeze timer. on_sneeze_callback can be used to do custom sneezes, on_sneeze_hit_callback for special effects, but probably usually making it infect
+/mob/living/proc/sneeze(range = 4, charge_time = 0.5 SECONDS, obj/projectile/sneezoid = /obj/projectile/sneeze, on_sneeze_callback = null, on_sneeze_hit_callback = null)
+	if(charge_time)
+		emote("inhale")
+
+	var/callback = on_sneeze_callback || CALLBACK(src, PROC_REF(launch_sneeze), range, sneezoid, on_sneeze_hit_callback)
+	addtimer(callback, charge_time)
+
+/// Shoot the sneeze projectile
+/mob/living/proc/launch_sneeze(range, obj/projectile/sneezoid, datum/callback/on_sneeze_hit_callback)
+	emote("sneeze")
+
+	var/obj/projectile/sneezium = new sneezoid(get_turf(src), on_sneeze_hit_callback)
+	sneezium.range = range
+	sneezium.firer = src
+	sneezium.fire(dir2angle(dir))
+
+/// Sneeze projectile launched by sneezing. gross
+/obj/projectile/sneeze
```

Comment:
Also IIRC the way you have set this up means it will consume shielded hardsuit charges & similar effects - `/obj/projectile/kiss` had to do some snowflake to get around that

Reply:
Unlike kisses, sneezes (this implementation) can't be forced. I think it's rare and immersive, and I also really don't like the workaround kisses use. I'll think on it

Reply:
I disagree. I'd rather not encourage spreading the common cold to combat shield components passively. It takes zero effort to spread the cold, it comes prepackaged in virology and does literally nothing harmful as a baseline.